CONTRIBUTION.md

```diff
@@ -58,7 +58,51 @@ Devenv will automatically give you access to all other relevant tools
 for a full list of installed tools have a look at the `devenv.nix` file.
 
 
+### Poetry-based installation (deprecated, broken)
+
+- [poetry](https://python-poetry.org/)
+- recommended:
+  - [pre-commit](https://pre-commit.com/)
+  - [node](https://github.com/nvm-sh/nvm)
+  - [commitlint](https://github.com/conventional-changelog/commitlint) to help following our [conventional commit]
+  (https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary) guidelines
+poetry can be installed in the following way:
+```bash
+pip install poetry
+poetry install
+poetry shell
+pre-commit install
+npm install --save-dev @commitlint/{config-conventional,cli}
+sudo apt install ghdl
+```
+
+The repository should work with the following versions:
+- Python: >=3.11
+- GHDL: >=1.0.0, <=4.1.0
+- Node: >=v12.22.9, <=v22.4.1
+
+
 ## Pull Requests and Commits
+
+### Conventional Commit Rules
+
+We use conventional commits (see [here](https://www.conventionalcommits.org/en/v1.0.0-beta.2/#summary)). The following commit types are allowed. The message scope is optional.
+
+| Commit Types |
```

Comment:
Tbh, i'd personally rather leave this out and concentrate on telling people to properly use feat/fix commit msg types. Imo, people should rather have a look at the two sections below than having to digest this table. I don't see the table being too useful without explaining the exact meaning of each of these types. That's why i propose to just stick with feat/fix and make the others optional. If the commit msg itself is clear, no one really needs the type, except for auto generating changelogs. What do you guys think?